src/style.css

```diff
@@ -1 +1,80 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  /* stylelint-disable-next-line string-quotes */
+  font-family: 'Roboto', sans-serif;
+  font-size: 500;
+
+}
+
+body {
+  margin: 0;
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.header_logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+nav {
+  display: block;
```

Comment:
The 'display: block;' is unnecessary for the 'nav' element, as it is a block element by default.